upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -144,3 +144,8 @@ There you can find links to upgrade notes for other versions too.
 
 - add additional data to Product frontend API type ([#2057](https://github.com/shopsys/shopsys/pull/2057))
     - see #project-base-diff to update your project
+
+- add promoted products in frontend API ([#2066](https://github.com/shopsys/shopsys/pull/2066))
+    - translation in the administration was changed from "Main page products" to "Promoted products"
+        - translation ID will be changed accordingly in the next major
```

Comment:
I am not really sure how we handle it, there is no deprecation sign. Should it be there?

Reply:
Part of removing deprecations before a major release is checking for changed translations in messages and update msgids accordingly, so all English translations have empty msgstr.

Reply:
Ok, thanks!